gammapy/scripts/image_pipe.py

```diff
@@ -21,37 +28,152 @@ def image_pipe_main(args=None):
 
 class ImageAnalysis(object):
     """Gammapy 2D image based analysis.
+
+    center : `astropy.coordinates.SkyCoord`
+        Center of the image
+    energy_band : `~gammapy.utils.energy.Energy
+        Energy band for which we want to compute the image
+    offset_band :`astropy.coordinates.Angle`
+        Offset Band where you compute the image
+    data_store : `~gammapy.data.DataStore`
+        `DataStore` where are situated the events
     """
 
-    def __init__(self, config):
-        self.config = config
-
-    @classmethod
-    def from_yaml(cls, filename):
```

Comment:
I think in the end we'd like to have config I/O for image analysis tasks, but as this currently doesn't do anything it's probably OK to remove it...